### DISCUSSION TITLE
Roll back library

### DISCUSSION THREAD

**Discussion by B1Gt0ne** [Bodmer/TFT_eSPI#2204] [Q&A - General] [answered]
I updated the TFT_Espi library this morning and now I cant get anything to display on my TFT screen. I have tried several previous versions of the library but still no joy.
The serial monitor shows that everything is running in the background but I have lost the display output.
I remember when I first had the TFT screen that I ran an example demo that displayed which TFT I was using etc but I cant seem to find this demo

**Comment by Bodmer** [comment, marked as answer]
When TFT_eSPI is updated the setup files are reset. So I suspect this is the problem. This part of the readme provides more information:
https://github.com/Bodmer/TFT_eSPI#tips

**Reply by B1Gt0ne** [reply]
Thank you Bodmer. It was the ILI9488_driver that had defaulted to the ILI9341_driver
